test/integration/api.bats

```diff
@@ -48,3 +48,50 @@ function teardown() {
        [ "${#lines[@]}" -eq  2 ]
        [[ "${lines[1]}" == *"false"* ]]
 }
+
+
+@test "docker create/start/stop/restart/kill should return success" {
+	start_docker 1
+	swarm_manage
+	
+	#create
+	run docker_swarm create --name test_container busybox sleep 1000
+	[ "$status" -eq 0 ]
+	run docker_swarm ps -l
+	[ "$status" -eq 0 ]
+	[ "${#lines[@]}" -eq 2 ]
+	[[ "${#lines[1]}" == *"test_container"* ]]
```

Comment:
You need to remove the `#` in front of `lines[1]` (for all the following tests too)